.github/actions/README.md

```
@@ -0,0 +1,154 @@
+
+#### Inputs
+
+| Input | Description | Required | Default |
+| :--- | :--- | :---: | :--- |
+| `working-directory` | The directory to install dependencies in (if `install-paths` is empty) | No | `"."` |
+| `install-paths` | Newline-separated paths to install dependencies in (relative to the workspace root) | No | `""` |
+| `install-args` | Newline-separated Bun install arguments | No | `""` |
+
+#### Usage Examples
+
+##### Single-directory install (current directory)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)